docs/src/pages/components/data-grid/columns/columns.md

```diff
@@ -41,6 +41,14 @@ You can configure the headers with:
 
 For more advanced header configuration, go to the [rendering section](/components/data-grid/rendering/#header-cell).
 
+## Column menu
+
+By default, for each column header a column menu is shown that allows filters to be set in the context of the target column. To disable it set the prop `disableColumnMenu={true}`.
```

Comment:
```suggestion
By default, each column header displays a column menu. The column menu allows actions to be performed in the context of the target column, e.g. filtering. To disable it set the prop `disableColumnMenu={true}`.
```

Reply:
Ok I was thinking about how to structure it because all but one function of the column menu is related to filtering but `actions` sounds more appropriate.